src/main/resources/jenkins/plugins/git/maintenance/MaintenanceUI/help-commit-graph.html

```diff
@@ -0,0 +1,3 @@
+<div>
```

Comment:
@MarkEWaite needs to find the technique to add the help icon to the user interface.  Users will expect to view help for the various maintenance commands.